x/datapool/types/keys.go

```diff
@@ -55,10 +62,16 @@ func GetKeyPrefixDataValidateCert(poolID, round uint64, dataHash []byte) []byte
 }
 
 func GetKeyPrefixNFTRedeemReceiptByPoolID(poolID uint64, redeemer sdk.AccAddress) []byte {
-	keyPoolAppended := append(KeyPrefixNFTRedeemReceipts, sdk.Uint64ToBigEndian(poolID)...)
-	return append(keyPoolAppended, redeemer.Bytes()...)
+	combineKeys := CombineKeys(sdk.Uint64ToBigEndian(poolID), redeemer)
+	return append(KeyPrefixNFTRedeemReceipts, combineKeys...)
+}
+
+func GetKeyPrefixNFTRedeemReceipt(poolID, round, nftID uint64, redeemer sdk.AccAddress) []byte {
```

Comment:
As we discussed off-line, let's use poolID, round, and nftID for the key of data pass redeem receipt.

Reply:
Fixed in [a64419f](https://github.com/medibloc/panacea-core/pull/322/commits/a64419f272d3bca82cc80d87de4dc39ab9f6c2b0). Thanks!